runtime/lua/vim/lsp/util.lua

```diff
@@ -684,6 +684,15 @@ function M.text_document_completion_list_to_complete_items(result, prefix)
   return matches
 end
 
+---@private
+local function bufwin(bufnr)
```

Comment:
can we use `vim.fn.bufwinid()` ?

Reply:
Turns out `bufwinid()` doesn't work across tabpages. Will need to revert to my original imp.